examples/medplum-chart-demo/src/components/clinical-chart/Medications.tsx

```diff
@@ -0,0 +1,85 @@
+import { Anchor, Badge, Box, Button, Group, Modal, Radio, Stack, Text } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import { getDisplayString } from '@medplum/core';
+import { CodeableConcept, MedicationRequest, Patient } from '@medplum/fhirtypes';
+import { CodeableConceptDisplay, CodeableConceptInput, Form, useMedplum } from '@medplum/react';
+import React, { useCallback, useState } from 'react';
+
+export interface MedicationsProps {
+  patient: Patient;
```

Comment:
Even though we're not using it yet, we've been adding `encounter?: Encounter;` as a prop in preparation for adding encounter support.  See `Allergies.tsx`.